include/aikido/control/ros/RosPositionCommandExecutor.hpp

```diff
@@ -0,0 +1,75 @@
+// Assume it has 4 positions
+#ifndef AIKIDO_CONTROL_ROS_ROSPOSITIONCOMMANDEXECUTOR_HPP_
+#define AIKIDO_CONTROL_ROS_ROSPOSITIONCOMMANDEXECUTOR_HPP_
+#include <chrono>
+#include <future>
+#include <mutex>
+#include <ros/ros.h>
+#include <ros/callback_queue.h>
+#include <sensor_msgs/JointState.h>
+#include <pr_control_msgs/SetPositionAction.h>
+#include <actionlib/client/action_client.h>
+#include <aikido/control/PositionCommandExecutor.hpp>
+#include <Eigen/Dense>
+
+namespace aikido{
+namespace control {
+namespace ros {
+
+/// This class sends commands for the hand position to the ROS server
```

Comment:
Nit: This comment is not very helpful. At a minimum, we should specify that this class uses `actionlib` to command a `pr_control_msgs/SetPosition` action.